index.php

```diff
@@ -2,7 +2,7 @@
 <html>
 	<head>
 		<meta charset="UTF-8" />
-
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/> <!--320-->

```

Comment:
Would prefer if the `maximum-scale` and `user-scalable` were removed. Users should be able to zoom into a page if needed.